magellan-legacy/src/main/java/com/wealthfront/magellan/HistoryRewriterExtensions.kt

```diff
@@ -24,8 +24,8 @@ internal fun HistoryRewriter.rewriteHistoryWithNavigationEvents(
     }
   }
   if (magellanTransition != null || navigationType != null) {
-    val lastNav = oldBackStack.last
-    oldBackStack.removeLast()
-    oldBackStack.add(NavigationEvent(lastNav.navigable, magellanTransition ?: navigationType!!.toTransition()))
+    val lastNav = oldBackStack.first
```

Comment:
sneaky fix

Reply:
Wouldn't the last nav be at the top of the stack? Maybe we should call pop() here for clarity (is `first` the top or bottom of the stack?)

Reply:
Done. Changed to pop.